content/en/docs/components/training/user-guides/mxnet.md

```diff
@@ -1,263 +0,0 @@
-+++
```

Comment:
Thanks for removing this!
Fixes: https://github.com/kubeflow/training-operator/issues/1996

Reply:
@andreyvelich Should we remove this document after we stop supporting the MXJob in the next release?
Indeed, users can use the MXJob in the latest TrainingOperator release.

Reply:
@tenzen-y Yeah, but the master branch representing the latest changes.
E.g. users can always use 1.9 branch if they want to get the release docs: https://v1-9-branch.kubeflow.org/

Reply:
But, we always expose and promote anywhere only the master branch website.
I guess that it's challenging to find location where there are MXJob docs, right?

Reply:
That's true, but how we can make sure if users deploy Training Operator from the master branch, they will not use the MXJob ?

Reply:
>Is the master branch ready for users?

I think, it depends on the user. If users are happy to use the development branch with the latest changes, they can always install from master. Also, we publish image to the DockerHub on each commit.

However, I am happy to keep the MXJob docs for sometime, if we feel that we should remove them once we make the first RC for the next release.
@kubeflow/wg-training-leads Any thoughts ?

Reply:
I can hold off on merging this PR until the next release if that's preferred @tenzen-y but my general recommendation is for us to not have to update the documentation twice needlessly. I can also just fork those changes in a separate PR. 

It's more work for probably not a ton of value but I'm happy to do it if that's what you all prefer.